docs/general/dql/selects.rst

```diff
@@ -841,6 +841,50 @@ Object property can also be addressed in the :ref:`where clause
     +------------+----------------------------------------------------------------------+
     SELECT 1 row in set (... sec)
 
+For selecting records having values with empty or unassigned objects, like::
+
+    cr> insert into locations (id, name, position, kind, inhabitants) values
+    ... (99, 'Voidspace', 0, 'Milliways', {});
+    INSERT OK, 1 row affected (... sec)
+
+.. Hidden: refresh locations
+
+    cr> refresh table locations;
+    REFRESH OK, 1 row affected (... sec)
+
+a suitable expression to match for both ``NULL`` values and empty objects, is::
+
+    cr> select count(*) from locations where inhabitants is null;
+    +----------+
+    | count(*) |
+    +----------+
+    |       12 |
+    +----------+
+    SELECT 1 row in set (... sec)
+
+.. Hidden: drop record again
+
+    cr> delete from locations where id=99;
+    DELETE OK, 1 row affected  (... sec)
+
+For determining the number of items in an object, you can use the
+:ref:`object_keys <scalar-object_keys>` and :ref:`array_length
+<scalar-array_length>` scalar functions::
+
+    cr> select count(*) from locations where
+    ... array_length(object_keys(inhabitants), 1) = 2;
+    +----------+
+    | count(*) |
+    +----------+
+    |        1 |
+    +----------+
+    SELECT 1 row in set (... sec)
```

Comment:
This section is now both nonsense and confusing. I will replace it with a better example to demonstrate the usage of the new `object_keys()` function.